src/Elmish.WPF/ViewModelUtilities.fs

```diff
@@ -0,0 +1,28 @@
+﻿[<RequireQualifiedAccess>]
+module Elmish.WPF.Utilities.ViewModel
```

Comment:
Is there any reason this (and the additions in the other coming PR from #52) is placed under `Elmish.WPF.Utilities` instead of `Elmish.WPF`? I'm not saying I disagree, I'm just wondering about the rationale behind it.

Reply:
Simply, I see my functions from #52 as utilities for advanced uses :) If you think it doesn't fit a good style for module naming, I'll retrieve that.

Reply:
Yes, I think you're right. Let's keep `Utilities`.